country-dashboard/src/Components/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Box, Image, Text, AspectRatio } from "@chakra-ui/react";
+
+function Card(props: any) {
+  return (
+    <Box
+      key={props.currency}
+      className="card--ui"
+      borderWidth="1px"
+      borderRadius="md"
+      overflow="hidden"
+      boxShadow="sm"
+      width="16.75rem"
```

Comment:
Don't hardcode width and height, you should be using flexbox properties or % instead. -3